packages/validator-ajv8/dts.config.js

```diff
@@ -0,0 +1,12 @@
+const replace = require('@rollup/plugin-replace');
+
+module.exports = {
+  // This function will run for each entry/format/env combination
+  rollup(config, options) {
+    //
+    if (options.format === 'cjs' && options.env === 'development') {
+      config.input['validator-ajv8'] = config.input['validator-ajv8'].replace('index.ts', 'index-dev.ts');
+    }
+    return config; // always return a config.
+  },
```

Comment:
I think this approach is going to cause more confusion, since one of the bundles will just be different. Could we just pass two files ([see docs](https://github.com/weiran-zsd/dts-cli#multiple-entry-files)) where one file is `index.ts` with `compileSchemaValidators` removed, as you have done, and then the other entrypoint could just be `compileSchemaValidators.ts`? Then in the docs I think we could then instruct users to precompile in Node by importing `@rjsf/validator-ajv8/compileSchemaValidators` (or maybe this would be `@rjsf/validator-ajv8/dist/esm/compileSchemaValidators` or `@rjsf/validator-ajv8/dist/cjs/compileSchemaValidators`, but whatever the end path is isn't that important)

Reply:
Putting the `compileSchemaValidators` into the esm build is part of the problem since that build is used by browsers because it is defined by `module` in the package.json. Since only cjs development ever needs the compiler then keeping it separate seems to be the best idea. Another option could be to provide a whole new package, `validator-ajv8-cli`, but then we'd have to do a better job of providing a CLI like `ajv-cli` does